CMakeLists.txt

```diff
@@ -561,8 +561,12 @@ if(${RUN_HIT} EQUAL 0)
     include_directories(${HIP_SRC_PATH}/tests/src)
     hit_add_directory_recursive(${HIP_SRC_PATH}/tests/src "directed_tests")
 
+    # Add unit tests
+    include_directories(${HIP_SRC_PATH}/tests/unittests)
```

Comment:
Naming the folder as tests/unittests sounds redundant. Maybe just go with tests/unit?

Reply:
Done